tools/copyright.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+# Copyright 2018 The Amber Authors.
```

Comment:
Switched this from Shaderc to Amber, given we have the copyright on both I figured that was OK?

Reply:
Yes, all contributions to that script in the Shaderc repo were from Googlers, so Google owns the copyright to the original contributions.